windows/src/test/unit-tests/kmcomp/test_casedkeys.kmn

```diff
@@ -0,0 +1,39 @@
+c Tests cased keys rule expansions
+store(&NAME) 'Test &CasedKeys'
+begin Unicode > use(main)
```

Comment:
Are there any tests against the rules themselves?  I'd like to see something for that if possible.

I can see that a compilation-success test runs, but nothing that compares evaluation of the keystrokes to their expected values.

Reply:
If there already are, and you can point me at them / the test results, that'd definitely expedite a LGTM.

Reply:
Currently, the test suite only tests for pass/fail. I would like to extend that to evaluating rule compilation but that's a lot harder. Even a compile+decompile is tricky:
* I can obviously do that with a fixture of the decompiled result
* But we'd need to manually check the fixture when we provide it, which is doable but not necessarily easy
* Version information is emitted in the decompiled keyboard, so we have to ignore that information when doing compares, so it's not a straight file comparison
* kmdecomp is itself a very rough tool, so guarantees on round-trippiness are a little vague.

So, if I do that, I'd like to do it as a separate PR, because it's going to add a bunch of plumbing. If I do that, I'd like to do the fixture for each of the existing tests, as well.